core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ClassFixtures.scala

```diff
@@ -37,12 +37,12 @@ trait RunningIntelliJPerSuiteBase {
   }
 
   def teardown(): Unit = {
-    try runningIntelliJFixture.foreach(r => AfterTestChecks(baseFixture.factory.config.check, r.probe))
+    try runningIntelliJFixture.foreach(r => AfterTestChecks(baseFixture.intelliJProvider.factory.config.check, r.probe))
     finally {
       try afterAll()
       finally {
         running.foreach(baseFixture.closeIntellij(_))
-        installed.foreach(baseFixture.deleteIntelliJ(_))
+        installed.foreach(baseFixture.cleanupIntelliJ(_))
```

Comment:
IDK if `cleanupIntelliJ` is a good place for it but I want to have a way to provide my own method that cleans up the instance (maybe something similar to `afterIntelliJInstall`, `afterWorkspaceSetup` or sth?)

Reply:
It might be worth creating a separate issue to limit the scope of this one. @lukaszwawrzyk WDYT?

Reply:
Sure it can be a separate PR. @MichalBednarz what do you need it for? 
We probably will add a method set like: beforeIJClose, beforeIJCleanup, beforeWorkspaceCleanup or afterIJClose, afterIJCleanup, afterWorkspaceCleanup. Not sure which make more sense, this is why I am wondering what do you need it for.

Reply:
@lukaszwawrzyk I want to be able to recreate the instance that is e.g. cleanup caches or totally wipe it out and reinstall.

Reply:
I think we should go with `after` variant maybe.